1-select-tasks/18.txt

```diff
@@ -1,3 +1,5 @@
 Create a SQL query to calcualte total orders count by each day in 1998:
 
```

Comment:
In general - ok. You can also use datepart with dayofyear. Dependent on requirements. 
https://docs.microsoft.com/en-us/sql/t-sql/functions/datepart-transact-sql?view=sql-server-ver15